outputs.tf

```diff
@@ -12,3 +19,15 @@ output "delegation_set" {
   description = "The outputs of the created delegation set."
   value       = try(aws_route53_delegation_set.delegation_set[0], null)
 }
+
+# ------------------------------------------------------------------------------
+# OUTPUT ALL INPUT VARIABLES AS-IS
```

Comment:
```suggestion
# OUTPUT ALL INPUT VARIABLES
```

Reply:
`AS-IS` is considered to be bad english. 

Let's use 

`# ALL INPUT VARIABLES`

or so instead?

Reply:
fixed in template